### DISCUSSION TITLE
Change footer contents

### DISCUSSION THREAD

**Discussion by JasonInOttawa** [johnfactotum/foliate#1303] [Q&A]
Is there any way to change what the footer is showing? 

My preference would be "pages in chapter" * "pages in book".

I've tried digging around in the menus and in the settings but haven't found anything.

Thanks!

**Comment by johnfactotum** [comment]
No, there's currently no way to change that in v3 yet, though the option existed in v2.

**Reply by JasonInOttawa** [reply]
Thanks
